### ISSUE TITLE
[Linux] desktop file is lacking a Categories= entry

### ISSUE THREAD

**Issue by probonopd** [nikolio/GradeStats#3]
aThe desktop file is lacking a `Categories=` entry in the `*.desktop` file.

According to the [menu spec](https://standards.freedesktop.org/menu-spec/latest/apa.html),

> By including one of the Main Categories in an application's desktop entry file, the application will be ensured that it will show up in a section of the application menu dedicated to this category. If multiple Main Categories are included in a single desktop entry file, the entry may appear more than once in the menu. 

Hence, please add at least one of the following in the `Categories=` key.

Main Category | Description | Notes
-- | -- | --
AudioVideo | Application for presenting, creating, or processing multimedia (audio/video) |  
Audio | An audio application | Desktop entry must include AudioVideo as well
Video | A video application | Desktop entry must include AudioVideo as well
Development | An application for development |  
Education | Educational software |  
Game | A game |  
Graphics | Application for viewing, creating, or processing graphics |  
Network | Network application such as a web browser |  
Office | An office type application |  
Science | Scientific software |  
Settings | Settings applications | Entries may appear in a separate menu or as part of a 		"Control Center"
System | System application, "System Tools" such as say a log viewer or network monitor |  
Utility | Small utility application, "Accessories"

In addition, you could specify one or more from the longer list of [Additional Categories](https://standards.freedesktop.org/menu-spec/latest/apas02.html).

Please test the result with `desktop-file-validate` and make sure it passes.

electron-builder offers native support for this since [v19.22.1](https://github.com/electron-userland/electron-builder/releases/tag/v19.22.1).

Reference: https://github.com/AppImage/AppImageHub/issues/2

**Comment by nikolio** [comment]
Is this issue opened by a bot?  Because it feels like it, no offense but is kinda strange opening the issue on a seemingly random repo. Anyway thanks for the response with all the info :)

**Comment by probonopd** [comment]
No, I am not a bot. Just trying to build a central directory of AppImages,

**Comment by nikolio** [comment]
Aha. Thanks for the info. The next push will contain a commit with a category in the package json. 
I might digress, but have you used recently electron builder with the win nsis, cause I cant seem to build the exe in Travis CI. Sth with nsis wanting to build with the new gcc abi that the trusty packages in Travis don't have.
Anyway
Thanks and good day
Nikos

**Comment by probonopd** [comment]
Hi Nikos, thank you very much. Unfortunately I don't know whether it is possible to build using electron-builder for Windows on Travis CI.

**Comment by probonopd** [comment]
Can you ping me once a new release is up for download? Thanks.

**Comment by nikolio** [comment]
Will do
Thanks

**Comment by nikolio** [comment]
https://github.com/nikolio/GradeStats/commit/7aa24a80599f7c63765a68e20b29ac3003aee08a adds a category option. I verified the desktop entry file with the desktop-file-validate
Release https://github.com/nikolio/GradeStats/releases/tag/0.0.2

Good luck with building the central directory :)
Nikos

**Comment by probonopd** [comment]
Thank you very much. Maybe you want to disable the developer tools by default? https://appimage.github.io/GradeStats/

**Comment by nikolio** [comment]
Well since the application is in heavy development I need them. I saw that is not pretty with the developer tools open. If I upload new release  the appimage hub entry will be updated automatically ?

**Comment by probonopd** [comment]
If you send a pull request on https://github.com/AppImage/appimage.github.io/blob/master/data/GradeStats (e.g., remove the `#`) then it will re-run the job and make another screenshot.

**Comment by nikolio** [comment]
Thanks I will make sure to send a pull request when I 'm gonna have a release with the dev tools closed
